src/pages/Settings.js

```diff
@@ -26,127 +38,104 @@ function Settings() {
     signOut(auth)
       .then(() => {
         // Sign-out successful.
-        navigate("/login");
+        navigate("/hci-final");
```

Comment:
should navigate to login bc they just signed out

Reply:
done